core/runtime/src/main/java/io/quarkus/runtime/annotations/ConfigDocEnum.java

```diff
@@ -0,0 +1,23 @@
+package io.quarkus.runtime.annotations;
+
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.Retention;
+import java.lang.annotation.Target;
+
+/**
+ * Provides a way to configure how an enum is handled.
+ */
+@Retention(RUNTIME)
+@Target({ FIELD })
```

Comment:
This should contain METHOD for sure. Not sure FIELD still makes sense?

Reply:
It works as expected with this change. Thanks @gsmet!

Reply:
Fixed it. I apparently need some rest :). Thanks for testing!